client/signup/controller.js

```diff
@@ -83,6 +91,7 @@ export default {
 			React.createElement( SignupComponent, {
 				path: context.path,
 				refParameter,
+				jetpackRedirect,
```

Comment:
That sounds like a good idea, and will prevent us from having to update the controller every time we want to check a different parameter in `Signup`. However, if we want to keep the current `refParameter` behavior, we'll need to save the initial value of `context.query` (in a new method like `saveQueryObject` that provides the same behavior as `saveRefParameter` for `context.query`) because the query string is replaced as soon as the `redirectToFlow` is hit.